### ISSUE TITLE
TypeError: can't convert nil into Integer

### ISSUE THREAD

**Issue by fbueno** [Strech/sidekiq-prometheus-exporter#16]
When adding the exporter as standalone service I get the following error when running `curl localhost:9292/metrics`

```
[2019-05-20 17:01:12] INFO  WEBrick::HTTPServer#start: pid=4744 port=9293
TypeError: can't convert nil into Integer
        (erb):11:in `format'
        (erb):11:in `to_s'
        /usr/share/ruby/erb.rb:876:in `eval'
        /usr/share/ruby/erb.rb:876:in `result'
        /usr/local/share/gems/gems/sidekiq-prometheus-exporter-0.1.11/lib/sidekiq/prometheus/exporter/standard.rb:26:in `to_s'
        /usr/local/share/gems/gems/sidekiq-prometheus-exporter-0.1.11/lib/sidekiq/prometheus/exporter/exporters.rb:38:in `block in to_s'
        /usr/local/share/gems/gems/sidekiq-prometheus-exporter-0.1.11/lib/sidekiq/prometheus/exporter/exporters.rb:38:in `map'
        /usr/local/share/gems/gems/sidekiq-prometheus-exporter-0.1.11/lib/sidekiq/prometheus/exporter/exporters.rb:38:in `to_s'
        /usr/local/share/gems/gems/sidekiq-prometheus-exporter-0.1.11/lib/sidekiq/prometheus/exporter.rb:48:in `call'
        /usr/local/share/gems/gems/rack-2.0.7/lib/rack/urlmap.rb:68:in `block in call'
        /usr/local/share/gems/gems/rack-2.0.7/lib/rack/urlmap.rb:53:in `each'
        /usr/local/share/gems/gems/rack-2.0.7/lib/rack/urlmap.rb:53:in `call'
        /usr/local/share/gems/gems/rack-2.0.7/lib/rack/tempfile_reaper.rb:15:in `call'
        /usr/local/share/gems/gems/rack-2.0.7/lib/rack/lint.rb:49:in `_call'
        /usr/local/share/gems/gems/rack-2.0.7/lib/rack/lint.rb:37:in `call'
        /usr/local/share/gems/gems/rack-2.0.7/lib/rack/show_exceptions.rb:23:in `call'
        /usr/local/share/gems/gems/rack-2.0.7/lib/rack/common_logger.rb:33:in `call'
        /usr/local/share/gems/gems/rack-2.0.7/lib/rack/chunked.rb:54:in `call'
        /usr/local/share/gems/gems/rack-2.0.7/lib/rack/content_length.rb:15:in `call'
        /usr/local/share/gems/gems/rack-2.0.7/lib/rack/handler/webrick.rb:86:in `service'
        /usr/share/ruby/webrick/httpserver.rb:140:in `service'
        /usr/share/ruby/webrick/httpserver.rb:96:in `run'
        /usr/share/ruby/webrick/server.rb:307:in `block in start_thread'
127.0.0.1 - - [20/May/2019:17:01:40 +0000] "GET /metrics HTTP/1.1" 500 63791 0.1312
```

```
 cat config.ru
require 'sidekiq'
require 'sidekiq/prometheus/exporter'

Sidekiq.configure_client do |config|
  config.redis = {url: 'redis://redis:6379'}
end

run Sidekiq::Prometheus::Exporter.to_app
```

To start it: `bundle exec rackup -p9292 -o0.0.0.0`

Any ideas?

I'm using enterprise sidekiq.

**Comment by Strech** [comment]
@fbueno hey 👋 thanks for the issue, I've noticed the same behaviour when Sidekiq never was launched and don't create any structure in the Redis. I will fix it today 🔍

**Comment by fbueno** [comment]
Ok. I think we can close this one then. I mean, that's not a real issue. Maybe a friendly error message?

**Comment by Strech** [comment]
@fbueno yes, maybe a good error message will be good, but still no error is better : laughing:

**Comment by Strech** [comment]
@fbueno I think it was a bug with Sidekiq empty worker stats, fixed with #17

**Comment by fbueno** [comment]
thanks.
